utility/src/main/java/org/oppia/util/parser/UrlImageParser.kt

```diff
@@ -60,7 +69,40 @@ class UrlImageParser private constructor(
           } else {
             0
           }
-          val rect = Rect(initialDrawableMargin, 0, drawableWidth + initialDrawableMargin, drawableHeight)
+          val rect =
+            Rect(initialDrawableMargin, 0, drawableWidth + initialDrawableMargin, drawableHeight)
+          drawable.bounds = rect
+          urlDrawable.bounds = rect
+          urlDrawable.drawable = drawable
+          htmlContentTextView.text = htmlContentTextView.text
+          htmlContentTextView.invalidate()
+        }
+      }
+    }
+  }
+
+  private inner class SvgBitmapTarget(private val urlDrawable: UrlDrawable) :
```

Comment:
Nit for formatting:

```kotlin
private inner class SvgBitmapTarget(
  private val urlDrawable: UrlDrawable
): CustomTarget<PictureDrawable>() {
  ...
}
```

I think that wraps a bit better, but I'm not sure how ktlint will handle this case.

Reply:
Done.